json_schema_fuzz.py

```diff
@@ -17,6 +18,17 @@ def random_object(schema):
             object[key] = generate_json(value)
     return object
 
+def random_boolean(schema):
+    """Generate random JSON boolean."""
+    return random.choice([True, False])
+
+def random_string(schema):
+    """Generate random string."""
+    lowercase_letters = string.ascii_lowercase
+    word_length = random.randrange(1, (20+1))
+    new_word_list = random.choices(lowercase_letters, k = word_length)
```

Comment:
I'm going to try to get some linting and testing running through GitHub Actions. In the meantime, let's try to adhere to pylint and pycodestyle (PEP8) as much as possible. Right here that means removing the spaces around the "=" for keyword arguments.